translations/demo_check.phrases.txt

```diff
@@ -35,7 +35,7 @@
     "kicked_announce_disabled"
     {
         "#format" "{1:s}"
-        "en"    "{red}Warning! {lime}{1}{white} does not appear to be auto-recording demos correctly."
+        "en"    "{red}Warning! {lime}{1}{white} does not appear to be auto-recording demos correctly. If this is a match, warnings must be disabled."
```

Comment:
I don't know if this is the right language to use - I cant speak for other leagues, but Ozfortress is yet to move on from warnings even for official matches due to the significant number of users who still depend on manual recording.

Perhaps better language could be something akin to "If this is a match, please ensure you are manually recording a demo."